packages/debugger/test/endstate.js

```diff
@@ -33,42 +33,51 @@ uint x;
 }
 `;
 
-let sources = {
+const sources = {
   "FailureTest.sol": __FAILURE,
   "SuccessTest.sol": __SUCCESS
 };
 
 describe("End State", function () {
-  var provider;
-
-  var abstractions;
-  var compilations;
+  let provider;
+  let abstractions;
+  let compilations;
 
   before("Create Provider", async function () {
-    provider = Ganache.provider({ seed: "debugger", gasLimit: 7000000 });
+    provider = Ganache.provider({
+      seed: "debugger",
+      gasLimit: 7000000,
+      logging: {
+        quiet: true
+      },
+      miner: {
+        instamine: "strict"
+      }
+    });
   });
 
   before("Prepare contracts and artifacts", async function () {
     this.timeout(30000);
 
-    let prepared = await prepareContracts(provider, sources);
+    const prepared = await prepareContracts(provider, sources);
     abstractions = prepared.abstractions;
     compilations = prepared.compilations;
   });
 
   it("correctly marks a failed transaction as failed", async function () {
-    let instance = await abstractions.FailureTest.deployed();
+    const instance = await abstractions.FailureTest.deployed();
     //HACK: because this transaction fails, we have to extract the hash from
     //the resulting exception (there is supposed to be a non-hacky way but it
     //does not presently work)
     let txHash;
     try {
-      await instance.run(); //this will throw because of the revert
+      await instance.run();
     } catch (error) {
-      txHash = error.hashes[0]; //it's the only hash involved
+      txHash = error.receipt.transactionHash;
     }
+    if (!txHash) assert.fail("it should have set the txHash");
```

Comment:
here's another reformat to `assert.isDefined(txHash, "should have errored and set txHash");` ?

Reply:
```suggestion
    assert.isDefined(txHash, "should have errored and set txHash");
```